base/docs/helpdb/Base.jl

```diff
@@ -644,9 +644,9 @@ julia> exp10(0.2)
 exp10
 
 """
-    &(x, y)
+    x & y
 
-Bitwise and.
+Bitwise and. Add parenthesis for function application form: `(&)(x, y)`
```

Comment:
I'm not a native English speaker (so I may be wrong), but shouldn't it be "parentheses" (plural)?